python/cuml/fil/fil.pyx

```diff
@@ -382,7 +390,7 @@ class ForestInference(Base):
         super(ForestInference, self).__init__(handle)
         self._impl = ForestInference_impl(self.handle)
 
-    def predict(self, X, preds=None):
+    def predict(self, X, preds=None, predict_proba=False):
```

Comment:
I think we shouldn't add `predict_proba` here. It's better to have this parameter in `predict()` in `ForestInference_impl` only, and call it from both `predict()` and `predict_proba()`.